pyproject.toml

```diff
@@ -11,6 +11,7 @@ streamlit = "^1.36.0"
 boto3 = "^1.34.135"
 pandas = "^2.2.2"
 plotly = "^5.22.0"
+github-api-toolkit = { git = "https://github.com/ONS-Innovation/github-api-package.git", tag = "v1.0.0" }
```

Comment:
This dependency isn't required here. The Poetry dependencies are required solely to run the dashboard, not the lambda. For Lambda dependencies, these are kept in `./aws_lambda/requirements.txt`. This is because the lambda requirements get installed through docker when containerising the lambda.

Reply:
My mistake, I used for testing then got muddled up.